packages/react-tv-cli/scripts/tizen/run.js

```diff
@@ -0,0 +1,177 @@
+const path = require('path');
+const fs = require('fs-extra');
+const chalk = require('chalk');
+const execSync = require('child_process').execSync;
+
+function defaultCLIEnv() {
+  //return '/opt/tizen/tools/ide/bin';
+  // return 'E:/Ferramentas/tizen/tools/ide/bin';
+  return '/Users/celio.latorraca/tizen-studio/tools/ide/bin';
+}
+
+function getPackageId(root) {
+  const appinfo = path.resolve(root, 'react-tv/tizen/config.xml');
+  const content = fs.readFileSync(appinfo, {encoding: 'utf-8'});
+
+  const re = new RegExp(/tizen:application id="(.*?)"/);
+  const matches = content.match(re);
+
+  if (!matches) {
+    return null;
+  }
+
+  return matches[1];
+}
+
+function isReactTVTizenProject(root) {
+  const appinfo = path.resolve(root, 'react-tv/tizen/config.xml');
+  if (fs.existsSync(appinfo)) {
+    return true;
+  }
+  return false;
+}
+
+function run(root) {
+  let tizen_CLI_ENV = process.env.TIZEN_CLI || false;
+  if (!tizen_CLI_ENV) {
```

Comment:
I need to think better about this approach. Because React-TV already tries to predict EnvPath on WebOS scenarios. I don't have assure if this is the best approach. 

Another idea is allow to set configuration for binary paths on `package.json`